docs/root/configuration/http/http_filters/jwt_authn_filter.rst

```diff
@@ -20,8 +20,6 @@ Following are supported JWT alg:
 Configuration
 -------------
 
-This filter should be configured with the name *envoy.filters.http.jwt_authn*.
```

Comment:
this one is missing any api link - which leaves the page a bit of a dead end in terms of configuration

Reply:
Yes, let's put back the type URLs where it's missing. The link is the type URL that should be used generally.